### PR TITLE
Fix import handler

### PR DESCRIPTION
**Pull request by Werkov**
ref #22 

(I attempted to include fixes + number in commit message to [automatically create the PR](http://stackoverflow.com/a/7189981/1351874), however, it didn't work (I did force push). So creating PR manually.

### CODE REVIEW

File: app/model/Events/Model/ImportHandler.php [hunk 1]
```diff
@@ -110,11 +113,16 @@ private function rowToValues($row) {
...
+            if (in_array($fieldName, $fieldNames) || $fieldName === $pkName) {
                 $fieldExists = true;
             }
+            if ($fieldName === $pkName && $pkName === $this->keyName) {
+                $pkExists = true;
+            }
         }
-        if (!$fieldExists) {
+
+        if (!$fieldExists || !$pkExists) {
```
Comment (timkol) [inline app/model/Events/Model/ImportHandler.php]: This fails when I want to join by non-PK column. e.g. name;status header in FoL.

File: app/model/Events/Model/ImportHandler.php [hunk 2]
```diff
@@ -110,11 +113,16 @@ private function rowToValues($row) {
                 $values[$baseHolderName] = new ArrayHash();
             }
             $values[$baseHolderName][$fieldName] = $value;
-            if (in_array($fieldName, $fieldNames)) {
+            /* $keyName can also be primary key, consider it an implicit field too */
+            if (in_array($fieldName, $fieldNames) || $fieldName === $pkName) {
                 $fieldExists = true;
             }
+            if ($fieldName === $pkName && $pkName === $this->keyName) {
```
Comment (timkol) [inline app/model/Events/Model/ImportHandler.php]: IMO unneccessary (whole thing with $pkExists).

File: .gitmodules [hunk 3]
```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/facebook/facebook-php-sdk.git
 [submodule "libs/libfks"]
 	path = libs/libfks
-	url = git@fykos.cz:libfks.git
+	url = https://github.com/fykosak/php-libfks.git
```
Comment (timkol) [inline .gitmodules]: In CSV parser L63 '/^.../' instead of '/.../' regexp should be safer.

**Comment by cevro** [comment]
Is this PR still actual?

**Comment by Werkov** [comment]
It's an attempted fix of #22, if the affected import works/became non-issue. This can be closed.

**Pull request closed without merging**